pluma/smclient/eggdesktopfile.c

```diff
@@ -946,6 +951,7 @@ start_startup_notification (GdkDisplay     *display,
 
   if (launch_time == (guint32)-1)
     launch_time = gdk_x11_display_get_user_time (display);
+
```

Comment:
Is there a x11-free equivalent of `gdk_x11_display_get_user_time()`? If not maybe we should make this entire function x11-only.

Reply:
`g_get_monotonic_time () / 1000` is quite close to `gdk_x11_display_get_user_time ()`
see https://github.com/mate-desktop/mate-terminal/pull/342#discussion_r409080735